src/index.html

```
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lift Simulator</title>
+    <link rel="stylesheet" href="./css/main.css">
+</head>
+<body>
+    <header>
+        <h1 class="welcome__message">welcome to Lift simulator</h1>
+      <section class="countShow_section">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </section> ], start tag match failed [ <section class="countShow_section"> ] on line 12.

(tag-pair)